moped-editor/src/views/projects/newProjectView/DefineProjectForm.js

```diff
@@ -25,20 +25,50 @@ const DefineProjectForm = ({
       <Grid container spacing={3} style={{ margin: 20 }}>
         <Grid item xs={6}>
           {!fromSignalAsset && (
-            <TextField
-              required
-              autoFocus
-              label="Project name"
-              name="project_name"
-              variant="standard"
-              type="text"
-              fullWidth
-              value={projectDetails.project_name}
-              error={nameError}
-              helperText="Required"
-              InputLabelProps={{ required: false }}
-              onChange={e => handleFieldChange(e.target.value, e.target.name)}
-            />
+            <>
+              <TextField
+                sx={{ marginBottom: "1.5rem" }}
+                required
+                autoFocus
+                label="Project name"
+                name="project_name"
+                type="text"
+                fullWidth
+                value={projectDetails.project_name}
+                error={nameError}
+                helperText="Required"
+                InputLabelProps={{ required: false }}
+                onChange={(e) =>
+                  handleFieldChange(e.target.value, e.target.name)
+                }
+              />
+              <TextField
+                sx={{ marginBottom: "2.5rem" }}
+                label="Secondary name"
+                name="project_name_secondary"
+                type="text"
+                fullWidth
+                value={projectDetails.project_name_secondary || ""}
+                onChange={(e) =>
+                  handleFieldChange(e.target.value, e.target.name)
+                }
+              />
+              <TextField
+                required
+                label="Description"
+                name="project_description"
+                multiline={true}
+                type="text"
+                fullWidth
+                value={projectDetails.project_description}
+                error={descriptionError}
+                helperText="Required"
+                InputLabelProps={{ required: false }}
+                onChange={(e) =>
+                  handleFieldChange(e.target.value, e.target.name)
+                }
+              />
```

Comment:
these need to be pulled out of this conditional, so that if a project is from a signal asset, we can still include a secondary name and a description

Reply:
Absolutely, thank for catching that. I'm going push up a fix for this this afternoon.

Reply:
Thanks again! I put this back together the correct way and also took the opportunity to do some more work to make it match the mockups closer -- this was done by having the `description` field be 5 lines tall at a minimum.